### ISSUE TITLE
pseudo-code: inconsistent syntax?

### ISSUE THREAD

**Issue by pjotrek-b** [FFmpeg/FFV1#147]
I'm not sure, but the pseudo-code parts seem to have correct, but inconsistent syntax/formatting...

For example here:
Why does "if (version <= 1)" have a "{ }", while "if (coder_type == 0)" doesn't?
Both contain the same number of lines inside the "if" block:

```c
Slice( ) {                                                    |
    if (version >= 3)                                         |
        SliceHeader( )                                        |
    SliceContent( )                                           |
    if (coder_type == 0)                                      |
        while (!byte_aligned())                               |
            padding                                           | u(1)
    if (version <= 1) {                                       |
        while (remaining_bits_in_bitstream( NumBytes ) != 0 ) |
            reserved                                          | u(1)
    }                                                         |
    if (version >= 3)                                         |
        SliceFooter( )                                        |
}                                                             |
```

Additionally, sometimes there are spaces before/after parentheses - sometimes not, sometimes mixed.
If that's not on purpose, may I go through the code snippets and normalize their formatting? :smile:

Yours,
Adrian Monk - eh, *Peter* :wink:

**Comment by JeromeMartinez** [comment]
It was not intended, I tried to keep consistency and have "{ }" everywhere, sometimes we missed to catch the inconsistency.
Go ahead (with "{ }").

**Comment by JeromeMartinez** [comment]
> Also for "one-liners", like the first one in the above example?

Yes.

**Comment by retokromer** [comment]
I was willing to make the pseudo-code consistent on the whole document…

**Comment by pjotrek-b** [comment]
@retokromer 
Now that PREFORMA's over, let me have some commits to the docs, too :stuck_out_tongue_winking_eye:

**Comment by pjotrek-b** [comment]
I've filed a pull-request for this :grin:

**Comment by retokromer** [comment]
Is this still an open issue, @pjotrek-b?

**Comment by pjotrek-b** [comment]
Sorry. Forgot to close this.
Fixed in PR #148 :smile:

@retokromer Thanks for checking!
